.github/workflows/auto-publish.yml

```diff
@@ -18,12 +18,12 @@ jobs:
        - name: Publish to CurseForge
          uses: Kir-Antipov/mc-publish@v3.3
          with:
+           name: ${{ steps.download-files.outputs.tag_name }} for 1.19.4
```

Comment:
Use variables here and for game versions pls
https://docs.github.com/en/actions/learn-github-actions/variables

I believe this syntax could be used to keep the variable in-file while still reusing it for multiple jobs:
https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values

Reply:
We could - but what I implemented works for multiple workflows and you don't have to change the file every time (because why have to change the file every time if that's what we have to do right now?)

https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository

Reply:
> because why have to change the file every time if that's what we have to do right now?

well, I figured it would be easier to change a file in git flow than find some obscure github setting :D
but I'll try with the github method first then